cucumber-jparest/src/test/java/uk/gov/homeoffice/digital/sas/cucumberjparest/utils/SerialisationUtilTest.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.homeoffice.digital.sas.cucumberjparest.utils;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class SerialisationUtilTest {
+
+    @ParameterizedTest
+    @MethodSource("stringToMapTestData")
+    void GIVEN_serialised_map_WHEN_converted_to_map_THEN_produces_the_expected_map(String serialisedMap, int expectedMapSize, Map<String, String> expectedMap) {
```

Comment:
This tests the method which only takes in the `serialisedMap` param and then indirectly tests the overloaded one with `,` and `=` .
But the overloaded method which can take any delimeters also needs a test as the caller can provide their own.

Reply:
As mentioned above this is used internally. Current use case only includes `,` and `=` . If this is to be extended then test scenarios should be added accordingly

Reply:
yes, resolving based on previous comment response.